### ISSUE TITLE
Godot doesn't start with Linux Lite (Ubuntu) and intel driver

### ISSUE THREAD

**Issue by dannygaray60** [godotengine/godot#48564]
**Godot version:**

3.3 (latest stable)

**OS/device including version:**

Linux Lite latest version (Ubuntu 20.04.02 lts derivated) Intel Graphics and Xfce environment

**Issue description:**

Godot doesn't start if I add `etc/X11/xorg.conf.d/20-intel-graphics.conf` file to resolve **screen tearing** with videos and godot games (no matter what vsync is on) the pc freezes for 1 minute and then godot closes.

But if I remove that intel config file godot starts normal but screen tearing came back

![e62fe43](https://user-images.githubusercontent.com/30391210/117548296-703da780-aff1-11eb-84a7-d58f084db883.jpg)

**Steps to reproduce:**
Add `etc/X11/xorg.conf.d/20-intel-graphics.conf` with this configuration:

Section "Device"
   Identifier  "Intel Graphics"
   Driver      "intel"
   Option      "TripleBuffer" "true"
   Option      "TearFree"     "true"
   Option      "DRI"          "false"
EndSection

**Minimal reproduction project:**

**Comment by dannygaray60** [comment]
Nevermind my bad the name of config file is incorrect

**Comment by Calinou** [comment]
> Linux Lite latest version (Ubuntu 20.04.02 lts derivated) Intel Graphics and Xfce environment

Which IGP model is this exactly, and what CPU model do you have?

**Comment by dannygaray60** [comment]
CPU: Intel pentium 4415u 2.30 ghz

The graphic card is intel hd 610

And this issue is only for godot games

**Comment by Calinou** [comment]
@dannygaray60 I wonder if https://github.com/godotengine/godot/pull/48555 could help with this. If you can build Godot from source, can you [build lawnjelly's `delta_sync` branch](https://github.com/lawnjelly/godot/tree/delta_sync) and enable the option as described in the pull request?

**Comment by lawnjelly** [comment]
> @dannygaray60 I wonder if #48555 could help with this. If you can build Godot from source, can you [build lawnjelly's `delta_sync` branch](https://github.com/lawnjelly/godot/tree/delta_sync) and enable the option as described in the pull request?

I don't think that will help, it only changes the positions of the objects you render, not the timing of when frames are presented.
